apps/api/src/scraper/scrapeURL/index.ts

```diff
@@ -390,6 +390,9 @@ export async function scrapeURL(
   options: ScrapeOptions,
   internalOptions: InternalOptions,
 ): Promise<ScrapeUrlResponse> {
+  if (options.formats.includes("changeTracking") && options.waitFor < 5000) {
+    options.waitFor = 5000;
+  }
```

Comment:
This shouldn't be in this file. Put it in v1 types in the refine layer, where the extract/json logic is.